script.js

```diff
@@ -7,3 +7,30 @@ function burgermenu() {
     x.style.display = "block";
   }
 }
+document.addEventListener("submit", burgermenu);
+
+// Search function
+// const API_Key = "8e4669c252cd8c46bd10e12b16db058a";
+
+function handleSubmit(e) {
+  let text = document.getElementById("searchText").value;
+  let imgElem = document.getElementById("imgElement");
+  let query =
+    "https://api.edamam.com/search?q=${text}&app_id=225b43f6&app_key=8ca7eadc17dbccc23e301761ab625c3e&from=0&to=3";
```

Comment:
Right now, this is just a regular string, the `${text}` won't be exchanged with your variable. For template strings, you need to use backticks `` ` `` instead of regular quotes.